ui/site/src/tvSingle.ts

```diff
@@ -0,0 +1,22 @@
+import * as xhr from 'common/xhr';
+import throttle from 'common/throttle';
+
+const setZen = throttle(1000, zen =>
```

Comment:
We should change `throttle` to `throttlePromiseDelay` as done in this recent PR: https://github.com/lichess-org/lila/commit/89a14ce96772b751e55a2d2c2bbccc8942b100eb#diff-9ab3db33425e78c3bbd45a75c22f9952ab1355796f2f2cf1c02359a0eb136df1L46-R52